templates/partials/answers/checkbox.html

```diff
@@ -1,16 +1,16 @@
 {% from "components/checkboxes/_macro.njk" import onsCheckboxes %}
 
 {% set config = {
-  "legend": question_title,
+  "legend": answer.label,
```

Comment:
I think `"dontWrap": not answer.label,` is also needed here, as is done in radio, date answers etc